content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -103,10 +103,11 @@ To prepare to build the engine and projects, choose one of the following build t
 1. Use CMake to build the engine as an SDK, the same as if you installed the engine from an installer tool. The following example shows the `profile` build configuration.
```

Comment:
this statement is no longer valid because the config now shows the 3 options.

Reply:
You do have a point.